src/software/ai/hl/stp/tactic/goalie_tactic.h

```diff
@@ -101,6 +103,9 @@ class GoalieTactic : public Tactic
     Team getFriendlyTeam() const;
     Team getEnemyTeam() const;
 
+    // Distance to chip the ball when trying to yeet it
+    static constexpr double YEET_CHIP_DISTANCE_METERS = 2.0;
```

Comment:
I noticed that this constant is defined in a bunch of other header files. are you expecting robots with different tactics to chip it at different distances?

Reply:
opened https://github.com/UBC-Thunderbots/Software/issues/1878 to deal with it